\documentclass[fleqn,10pt]{wlscirep}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{lineno}
\usepackage{newfloat}
\usepackage[normalem]{ulem}
\useunder{\uline}{\ul}{}
\usepackage{lscape}
\DeclareFloatingEnvironment[name={Supplementary Figure},fileext=lsf,listname={List of Supplementary Figures}]{suppfigure}
\linenumbers

\title{Insights into biotic and abiotic modulation of the planktonic mesopelagic community in the Oceans}

\author[1,$\dag$]{Janaina Rigonato}
\author[2,$\dag$]{Marko Budinich}
\author[1,2]{\textit{et al.}}
%\author[1,*]{Olivier Jaillon}
%\affil[1]{Genoscope, department, Evry, postcode, country}
%\affil[2]{GO-SEE, department, Roscoff, postcode, country}

%\affil[*]{corresponding author(s): Olivier Jaillon (corresponding.author@email.example)}

\affil[$\dag$]{these authors contributed equally to this work}


\begin{abstract}
Extended Abstract Version

\textbf{Keywords}: Mesopepalgic plankton; OMZ; Tara Oceans; community ecology; 

\end{abstract}

\begin{document}
\flushbottom
\maketitle
%  Click the title above to edit the author information and abstract

\thispagestyle{empty}



\section*{Background \& Summary}

By definition, the mesopelagic zone is commonly delimited between 200–1,000 m depth below the ocean surface. Also, the low incidence of light in the water column, limiting the photosynthetic metabolism, until its total absence, where hunting is no more realized by visual search, can be considered the top-down biological limits to the mesopelagic layer, though also called twilight zone \cite{robinson_mesopelagic_2010}.

Mesopelagic zones with low oxygen concentration, called oxygen minimum zones (OMZ), are attracting the scientific community's interests since that there are indications that these zones are increasing in volume in the oceans. Understanding the communities dynamic in these regions can help predict possible impacts faced by global warming changes.

Previous reports showed the stratification of the planktonic communities in water layers with taxonomy changing with depth, where the mesopelagic zone upholds a distinct assemblage  of dsDNA viruses \cite{gregory_marine_2019}, giant viruses \cite{endo_biogeography_2020}, prokaryotes \cite{sunagawa_structure_2015,salazar_gene_2019} and eukaryotes \cite{giner_marked_2020}. Unlike epipelagic, the mesopelagic diversity does not follow the latitudinal diversity gradient trends from pole-to-pole increasing towards the equator \cite{ibarbalz_global_2019}.


Although the number of oceanic large scale surveys has increased in the last decade \cite{rusch_sorcerer_2007,karsenti_holistic_2011,pernice_global_2015} most studies in mesopelagic have been accomplished in geographically or ecologically fragmented sampling mainly due to difficulties of accessing this zone \cite{hidalgo_developing_2019}, which gives a fractioned idea of the community. Moreover, the factors influencing community structure presumably outcome from a combination of many biotic and abiotic factors \cite{lima-mendez_determinants_2015, louca_integrating_2016}, but the complex interplay among these factors remains barely known.

The present study takes advantage of the Tara Oceans expedition large-scale survey conducted in different water layers in a systematic sampling protocol, spanning from viruses to small eukaryotes size fractions, to investigate the mesopelagic complex biome. We capitalized on the produced data, the water physicochemical properties, and ocean geography to explore the mesopelagic community's differences compared to upper layers structuring. We also investigate possible water deoxygenation effects on the mesopelagic community by comparing oxygen minimum zones communities with those from well-oxygenated waters. This work expands our knowledge about the web of a relationship forming a mesopelagic ecosystem on a broad geographically scale.

\section*{Results}

In this study, we analyzed the community of 32 stations sampled in epipelagic and mesopelagic waters, including 13 samples characterized by a low concentration of oxygen (OMZ). Our data set was composed of metagenomic recruited viruses (NCDLV giant viruses - hereafter named as giruses and two dsDNA-viruses families Podoviridae and Mioviridae - hereafter named as viruses) and 16S-rRNA prokaryotes and V9 18S-rRNA metabarcoding pico-eukaryotes (0.8-5/0.8-3 $\mu$m).

\textit{As previously reported, we observed the epi/mesopelagic community stratification according to water column depth for all the assemblies evaluated (dsDNA-viruses, giruses, prokaryotes, and pico-eukaryotes).}  (\textbf{Supplementary Figure \ref{fig:nmds}}). %\textbf{Maybe this paragraph is not useful anymore, all this info is already published}

Our first goal was to understand the differences between epipelagic and mesopelagic beta-diversity variation that can be explained by environmental variables such as temperature, oxygen, salinity, NO$_3$, chlorophyll-a and particle flux (UVP). These variables are currently indicated as meaningful in oceanography. Firstly, we compared epipelagic and mesopelagic sampling sites employing their physicochemical properties, using Euclidian distance. We observed an elevate dissimilarity gradient among sites for both layers. Mesopelagic sampling points were spread in the plot, with most of the points placed distant from the group centroid (located in the center of the clouds of points identified for each group). While epipelagic points displayed a large variance due to few points positioned apart from the central cluster. %Despite the lower gradient range of the environmental variables in mesopelagic compared to the epipelagic sites, for instance temperature (~4$^{\circ}$C - ~18$^{\circ}$C for Mesopelagic and ~14$^{\circ}$C- ~27$^{\circ}$C for Epipelagic),
These results confirm the heterogeneity of environmental conditions composing both sampled layers. This environmental variation can directly reflect on the community composition.  (\textbf{Supplementary Figure \ref{fig:betadipersion}})

Variations in the community composition are always linked to four main processes: selection, dispersal, drift, and speciation \cite{vellend_conceptual_2010}, and two classes of ecological models, niche-based and neutral models, are frequently used to explain the assembly of communities in the most diverse environments (terrestrial and aquatic) \cite{chave_neutral_2004, mcgill_empirical_2006}. Our results showed that both epipelagic and mesopelagic communities fitted to niche/deterministic ecological models, selected by the lower AIC value (\textbf{Supplementary Electronic Material Table 1}). The great majority of both epipelagic and mesopelagic samples better fitted the lognormal model. This model has two fitted parameters, and one theory is that species are affected by several environmental and biotic competitive interactions \cite{wilson_methods_1991}. 

We could observe that the environment can explain a small fraction of the communities’ variance for both layers by applying a canonical correspondence analysis (32\% in average) (\textbf{Figure \ref{fig:cca_OS}}). The virus assembly was the exception, which about 54.9\% of the epipelagic variation, and 65.2\% of mesopelagic variation can be explained by the environmental factors included in the analysis (\textbf{Figure \ref{fig:cca_OS}}). Next, we tested the variance explained by each explanatory variable with all the others used as covariables (independently from their order in the model), but, differently from epipelagic communities, which is mainly governed by temperature and oxygen gradients as previously reported \cite{sunagawa_structure_2015,gregory_marine_2019,ibarbalz_global_2019,giner_marked_2020, ghiglione_pole--pole_2012}, we could not identify a common environmental predictor structuring all the mesopelagic assemblies. However, few different variables appeared as significant for different groups (\textbf{Table \ref{tab:CCA}, complete analysis in Supplementary Electronic Material Table 2}). More specifically, the oxygen in the mesopelagic layer was the main driver to explain the viruses and, as previously reported, the prokaryotes assemblies variation \cite{wright_microbial_2012, ulloa_pelagic_2013, aldunate_oxygen_2018}. 

On the other hand, even though we observed in the ordination plots the distinction of OMZ and oxic mesopelagic (Oxic\_meso) stations also for giruses and eukaryotes assemblies (\textbf{Figure \ref{fig:cca_OS}} diamond and inverse triangle), we could not disentangle the effect of the oxygen from the others variables included in the analyses (\textbf{Table \ref{tab:CCA}, complete analysis in Supplementary Electronic Material Table 2}). This result shows that these assemblies are probably affected equally by all the predictors evaluated, coping a larger environmental gradient that maximizes their niche-space partitioning. Previous studies had addressed the oxygen as one of the main drivers of eukaryotic communities in OMZ regions \cite{de_la_iglesia_distinct_2020, orsi_effect_2012, parris_microbial_2014}. These studies mainly compared the community composition along the oxygen gradient existing in the water column depth, from the surface downwards. However, it is clear the depth stratification of the planktonic community even in regions with high oxygen concentration, so depth must be taken into account in addition to the oxygen gradient in such cases \cite{schnetzer_depth_2011}.

The particle flux is a significant variable structuring the epipelagic and mesopelagic virus’ assemblies (\textbf{Table \ref{tab:CCA}, complete analysis in Supplementary Electronic Material Table 2}). This data supports previous reports about the high correlation of this environmental factor and viruses accounting for the carbon pump in epipelagic layers \cite{guidi_plankton_2016}, or their association suggesting the increase in the viruses input overlying water column via sedimenting particles \cite{parada_viral_2007}.

The heterogeneity in mesopelagic layers given by deep currents, shear zones, intertidal tides, and eddies may favor a patchy diversification in the mesopelagic community favoring species adaptation-acclimation. To further investigate this hypothesis, we identified nine different water masses at the mesopelagic sampled locations (\textbf{Supplementary Figure \ref{fig:T_Splot}}). Our results confirmed the water masses as a critical component of the mesopelagic communities variation for all the assemblies studied (viruses, giruses, prokaryotes and eukaryotes) based on permutation multivariate analysis of variance (\textbf{Table \ref{tab:PWM}}).

Another goal that we addressed here was to resolve planktonic community signatures of Oxic\_meso and OMZs regions contrasting to epipelagic ones. For this, we classified OTU’s based on their abundance into three eco-regions: epipelagic, Oxic\_meso and OMZ. The taxa that were equally abundant in all the eco-regions were classified as ubiquitous, and when present in Oxic\_meso and OMZ samples classified as core-meso. We obtained strong evidence for eco-regions taxonomical signatures by applying a robust methodology based on the statistical significance of relative OTUs abundances (\textbf{Supplementary Figure \ref{fig:sim_biome}}).

The great majority of viruses taxa occurred either in all regions (ubiquitous) or specific to the mesopelagic layer (\textbf{Supplementary Figures \ref{fig:sim_biome}, \ref{fig:tax_virus_heatmap}, \ref{fig:box_plot_assig}B}).This result suggests that viruses assembly's differences in water column might be due to the taxa adaptation to the mesopelagic environment. Two hypotheses arise here, 1) the environment act as a strong driver, directly selecting viruses independently of their hosts, and 2) there is a higher specificity in the relation virus-hosts in the mesopelagic layer. Firstly, if we consider the higher proportion of the viruses variation explained by the environmental predictors, compared to its prokaryotic hosts, we can posit that contrarily of what suggested the environment can directly impact viruses assembly composition governing their diversity. However, the increase in the enrichment of virus-hosts mesopelagic OTUs, especially in OMZ regions, does not discharge the virus-host indirect selection relationship.

While we found less specific giruses but more abundant OTUs in the mesopelagic samples (OMZ and Oxic-meso) (\textbf{Supplementary Figure \ref{fig:sim_biome}, \ref{fig:tax_virus_heatmap},\ref{fig:box_plot_assig}C}). Suggesting that giruses may infect a broader range of hosts thriving mesopelagic layer.

We were also able to identify enriched prokaryotic and eukaryotic taxa in the different eco-regions (\textbf{Supplementary Figure \ref{fig:sim_biome}}), in these cases, with a higher distinction between epipelagic and mesopelagic. An interesting finding was that the Oxic-meso and OMZ signatures were obtained mainly at the infra taxonomic level (OTUs) (\textbf{Figure \ref{fig:tax_trees}, Supplementary Electronic Material Tables 3 and 4}). In this case, we observed OTUs belonging to the same high taxonomic levels occurring in both Oxic-meso and OMZ samples (prokaryotes: Actinobacteria, Alphaproteobacteria, Thaumarchaeota; eukaryotes: Acantharea, Ciliophora, Diplonemida, MALV, RAD-B and Spumellaria). However, we can observe that different OTUs of the same taxonomic-group have preferences for one or other eco-region, being the majority non classified in infra-taxonomic level (\textbf{Supplementary Figures \ref{fig:box_plot_assig}, \ref{fig:tax_trees_sup}, Supplementary Electronic Material Tables 3 and 4}). This observation reflects our ignorance about the biodiversity and functional plasticity inherent of species thriving in this ecosystem.

Another step to better understand the dynamic of mesopelagic communities goes towards the ecological relationships among species that inhabit this layer. Co-occurrence networks indicate how the environment may structure the community composition and give us glimpses of ecological interactions among organisms. We derived a network of 6154 nodes and 12935 edges that presents associations between OTUs from giruses, prokaryotes and eukaryotes (\textbf{Figure \ref{fig:networks}A}). Due to the lower number of stations sampled for viruses, we excluded this group from this analysis. We run a module detection algorithm and we found three statistically significant modules that were composed of OTUs mainly enriched in mesopelagic samples together with ubiquitous OTUs (Oxic-meso module 1 and OMZ modules 4 and 17; \textbf{Figure \ref{fig:networks}B-D}).

The OMZ modules were composed of few connected nodes (323 and 175 nodes modules 4 and 17, respectively). Module 4 comprises the main taxa determined as OMZ signatures, especially for prokaryotes (Nitrospinae, Marinimicrobia SAR 406, Planctomycetes/Candidatus Scalindua) (\textbf{Figure \ref{fig:networks}C and D}). While the module 17 contains mainly eukaryotes interactions, for instance, MALV I and II described as parasites (\textbf{Figure \ref{fig:networks}C and D}). Giruses from the Megaviridae family are numerous in all the three mesopelagic modules. Megaviridae is a very abundant family in the oceans, and they infect eukaryotic communities of various sizes ranges from piconanoplankton (0.8-5 $\mu m$) up to mesoplankton (180-2,000 $\mu m$) \cite{mihara_taxon_2018, endo_biogeography_2020}. Unlike the OMZ modules, the Oxic\_meso module 1 showed more connections among taxa (731 nodes), a higher number of taxonomic-groups and a more equitable contribution of all the three assemblies evaluated (prokaryotes, eukaryotes and giruses) (\textbf{Figure \ref{fig:networks}C and D}). 

The OMZ and Oxic\_meso networks' differences suggest the loss of connections and interactions among the OMZ communities' members. That can directly affect ecosystem stability, which can promote habitat loss, endangering the environmental sustainability.

The structuring of mesopelagic communities does not follow the same rules as those reported for epipelagic. Consequently, variations on the same environmental predictors may cause different responses in the photic and aphotic communities. Besides, each mesopelagic assemblage has its particular behavior face to the environment predictors variations, increasing the challenge to predict consequences caused by drastic environmental changes. In this sense, we cannot extrapolate predictions derived from models generated by epipelagic to deep layers.

\newpage
\section*{Figures}

\begin{figure}[ht]
    \centering
    \includegraphics[scale=0.28]{images/custom_cca_plot_hellinger_no_bathy_labels_OS_regions_colors_to_print.pdf}
    \caption{Ordination plot of epipelagic and mesopelagic communities based on OTU’s composition based on canonical correspondence analysis (CCA), constrained by selected explanatory abiotic environmental parameters. IO: Indian Ocean, NAO: North Atlantic Ocean, NPO: North Pacific Ocean, SAO: South Atlantic Ocean, SPO: South Pacific Ocean}
    \label{fig:cca_OS}
\end{figure}
\clearpage
\begin{figure}[ht]
    \centering
    \includegraphics[scale=0.7,angle=90,origin=c]{images/hmap_general_pub.pdf}
    \caption{Taxonomic clustering of the Archaea, Bacteria and Eukarya Domains showing branches that are enriched in the different classes formed based on eco-regions OTUs assignation. UBI: ubiquitous, EPI: epipelagic, MES-OMZ: core mesopelagic, MES: Oxic\_meso, OMZ: oxygen minimum zone. Color range indicates the OTU abundance 0 to 1 normalized and size of node indicates OTU counting per node}
    \label{fig:tax_trees}
\end{figure}
\clearpage
\begin{figure}[ht]
    \centering
    \includegraphics[scale=0.5]{images/Networks_Composite_v3.pdf}
    \caption{Co-Ocurrence Network between Epipelagic and Mesopelagic depth. A) Total Network, with connected modules for OMZ (purple and orange) and MES (green) highlighted. B) Relative taxa abundance in each Module in each Station and depth. C) Relative number of OTUs classified in taxonomic groups. D) Network representation of modules enriched in Oxic\_meso and OMZ OTUS}
    \label{fig:networks}
\end{figure}
\clearpage

\section*{Tables}

\begin{table}[ht]
\centering
\caption{\label{tab:CCA} ANOVA p\_value for each environmental factor used as explanatory variable. Global correspond to the p-value for the whole set of variables, the following columns to p-values for each of the environmental variables considering the others as covariable.}
\begin{tabular}{llrrrrrrr}
\hline
 Assembly& Depth & Global & Temp. $^\circ$C & Salinity & O$_2$ [$\mu$mol/kg] &[NO3]$^-$  [$\mu$mol/l]	& Chl-a [mg/m$^3$] &	 Particle flux\\
\hline
\hline
Viruses & EPI & 0.001 & 0.057 & 0.144 & 0.043 & 0.031 & 0.420 & 0.017\\
Viruses & MES & 0.001 & 0.059 & 0.004 & 0.027 & 0.124 & 0.110 & 0.002\\
Giruses & EPI & 0.001 & 0.023 & 0.074 & 0.051 & 0.152 & 0.053 & 0.221\\
Giruses & MES & 0.002 & 0.032 & 0.211 & 0.145 & 0.134 & 0.008 & 0.211\\
Prokaryotes & EPI & 0.035 & 0.048 & 0.141 & 0.002 & 0.006 & 0.044 & 0.568\\
Prokaryotes & MES & 0.006 & 0.501 & 0.304 & 0.039 & 0.444 & 0.966 & 0.486\\
Eukaryotes & EPI & 0.025 & 0.027 & 0.166 & 0.412 & 0.292 & 0.216 & 0.659\\
Eukaryotes & MES & 0.001 & 0.606 & 0.191 & 0.243 & 0.468 & 0.271 & 0.477\\
\hline
\hline
\end{tabular}
\end{table}

\begin{table}[ht]
\centering
\caption{\label{tab:PWM} Proportion of the variation in community composition that explained by water masses using the Permutation multivariated analysis of variance (PERMANOVA)}
\begin{tabular}{lrrrrrr}
\hline
Assembly & Df & Sum Of Squares & Mean Squares & F. Model & R$^2$ & Pr(>F) \\
\hline
\hline
Viruses	& 4	& 1.87 &	0.47 & 2.32 &	0.51 &	0.002\\
Giruses	& 8	& 3.19 &	0.40 & 1.81 &	0.46 &	0.001\\
Prokaryotes & 8	& 1.30 &	0.16 & 3.29 &	0.60 &	0.001\\
Eukaryotes	& 8	& 2.60 &	0.32 & 1.62 &	0.36 &	0.001\\
\hline
\hline
\end{tabular}
\end{table}



Supplementary Electronic Material: \href{https://docs.google.com/spreadsheets/d/1N1xzjx8YBxD9Trgy7jHuE1G6fmDtgE7f3KkcxjNAJyc}{https://docs.google.com/spreadsheets/d/1N1xzjx8YBxD9Trgy7jHuE1G6fmDtgE7f3KkcxjNAJyc}

\clearpage

\section*{Supplementary Figures}

\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.4]{images/nmds_used_SF_to_print.pdf}
    \caption{Non-metric multidimensional scaling (NMDS) showing epipelagic and mesopelagic communities stratification for each organism group}
    \label{fig:nmds}
\end{suppfigure}
%\clearpage
\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.5]{images/betadisp_diganose_to_print.pdf}
    \caption{Epipelagic and mesopelagic group dispersion based on physical-chemical oceanic properties (Euclidian method). A) First two axes of PCoA. B) Dispersion of distances from samples to centroids.}
    \label{fig:betadipersion}
\end{suppfigure}
\clearpage
\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.2]{images/T_Splot.pdf}
    \caption{Temperature and salinity plot indicating water masses designation for all mesopelagic samples. Formats represent the different oceanic basins ( $\Box$- North Atlantic Ocean, $\bigcirc$  - South Atlantic Ocean, $\bigtriangleup$ - Pacific Ocean, $\bigstar$ - Indian Ocean). Colours indicate the oxygen concentration at the sampling depth. LSW - Labrador Sea Water; AAIW - Antarctic Intermediate Water; tNPIW – transitional North Pacific Intermediate Water; SAMW - Subantarctic Mode Water; SPSTMW - South Pacific Subtropical Mode Water; modAAIW - modified Antarctic Intermediate Water; PGW - Persian Gulf Water mass; RSW - Red Sea Water mass; NASTMW - North Atlantic Subtropical Mode Water.}
    \label{fig:T_Splot}
\end{suppfigure}
\clearpage
\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.9]{images/simple_biome_projection.pdf}
    \caption{Relative abundance of OTUs classified into different eco-regions in to ocean layers}
    \label{fig:sim_biome}
\end{suppfigure}

\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.4]{images/heatmap_Vir_Gir.pdf}
    \caption{Viruses and Giruses relative abundance enriched in each eco-regions class (UBI: ubiquitous, EPI: epipelagic, MES-OMZ: core mesopelagic, MES: Oxic\_meso, OMZ: oxygen minimum zone}
    \label{fig:tax_virus_heatmap}
\end{suppfigure}

\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.74,angle=270,origin=c]{images/box_plots_assign.pdf}
    \caption{Relative abundances of OTUs from taxonomical groups for epipelagic and mesopelagic (Oxic\_meso and OMZ) samples enriched in each eco-regions class (UBI: ubiquitous, EPI: epipelagic, MES-OMZ: core mesopelagic, MES: Oxic\_meso, OMZ: oxygen minimum zone. A) Prokaryotes B) Virus C) Girus D) Eukaryotes}
    \label{fig:box_plot_assig}
\end{suppfigure}

\begin{suppfigure}[ht]
    \centering
    \includegraphics[scale=0.6, origin=c]{images/Trees_OTU_sup_v2.pdf}
    \caption{Hierarchical clustering of Prokaryotic and Eukaryotic OTUs present in the different eco-regions. Ubiquitous taxa were removed from this representation. Full legend for annotated leaves are in \href{https://docs.google.com/spreadsheets/d/1N1xzjx8YBxD9Trgy7jHuE1G6fmDtgE7f3KkcxjNAJyc/edit?usp=sharing}{Supplemental Electronic Material}}
    \label{fig:tax_trees_sup}
\end{suppfigure}
\clearpage

%\subsection*{Subsection}
%
%Example text under a subsection. Bulleted lists may be used where appropriate, e.g.
%
%\begin{itemize}
%\item First item
%\item Second item
%\end{itemize}
%
%\subsubsection*{Third-level section}
% 
%Topical subheadings are allowed.
%
%\section*{Data Records}
%
%The Data Records section should be used to explain each data record associated with this work, including the repository where this information is stored, and to provide an overview of the data files and their formats. Each external data record should be cited numerically in the text of this section, for example \cite{Hao:gidmaps:2014}, and included in the main reference list as described below. A data citation should also be placed in the subsection of the Methods containing the data-collection or analytical procedure(s) used to derive the corresponding record. Providing a direct link to the dataset may also be helpful to readers (\hyperlink{https://doi.org/10.6084/m9.figshare.853801}{https://doi.org/10.6084/m9.figshare.853801}).
%
%Tables should be used to support the data records, and should clearly indicate the samples and subjects (study inputs), their provenance, and the experimental manipulations performed on each (please see 'Tables' below). They should also specify the data output resulting from each data-collection or analytical step, should these form part of the archived record.
%
%\section*{Technical Validation}
%
%This section presents any experiments or analyses that are needed to support the technical quality of the dataset. This section may be supported by figures and tables, as needed. This is a required section; authors must present information justifying the reliability of their data.
%
%\section*{Usage Notes}
%
%The Usage Notes should contain brief instructions to assist other researchers with reuse of the data. This may include discussion of software packages that are suitable for analysing the assay data files, suggested downstream processing steps (e.g. normalization, etc.), or tips for integrating or comparing the data records with other datasets. Authors are encouraged to provide code, programs or data-processing workflows if they may help others understand or use the data. Please see our code availability policy for advice on supplying custom code alongside Data Descriptor manuscripts.
%
%For studies involving privacy or safety controls on public access to the data, this section should describe in detail these controls, including how authors can apply to access the data, what criteria will be used to determine who may access the data, and any limitations on data use. 
%
%\section*{Code availability}
%
%For all studies using custom code in the generation or processing of datasets, a statement must be included under the heading "Code availability", indicating whether and how the code can be accessed, including any restrictions to access. This section should also include information on the versions of any software used, if relevant, and any specific variables or parameters used to generate, test, or process the current dataset. 



\bibliography{MesoLibrary_2}

\section*{Acknowledgements} %(not compulsory)

Acknowledgements should be brief, and should not include thanks to anonymous referees and editors, or effusive comments. Grant or contribution numbers may be acknowledged.

This study is part of the “Ocean Plankton, Climate and Development” project funded by the French Facility for Global Environment (FFEM). Rigonato J., Budinich B., Murillo A.A., Pierella Karlusich J.J., Brandão M.C. and Soviadan Y.D. received financial support from FFEM to execute the project. Brandão, M.C. also received financial support from Coordination for the Improvement of Higher Education Personnel of Brazil (CAPES 99999.000487/2016-03). We acknowledge Noan Le Bescot (Ternog Design) for assistance in preparing figures. Tara Oceans (which includes both the Tara Oceans and Tara Oceans Polar Circle expeditions) would not exist without the leadership of the Tara Expeditions Foundation and the continuous support of 23 institutes (http://oceans.taraexpeditions.org). We further thank the commitment of the following sponsors: CNRS (in particular Groupement de Recherche GDR3280 and the Research Federation for the study of Global Ocean Systems Ecology and Evolution, FR2022/Tara Oceans-GOSEE), European Molecular Biology Laboratory (EMBL), Genoscope/CEA, The French Ministry of Research, and the French Government ‘Investissements d’Avenir’ programmes OCEANOMICS (ANR-11-BTBR-0008), FRANCE GENOMIQUE (ANR-10-INBS-09-08), MEMO LIFE (ANR-10-LABX-54), and PSL* Research University (ANR-11-IDEX-0001-02). We also thank the support and commitment of agnès b. and Etienne Bourgois, the Prince Albert II de Monaco Foundation, the Veolia Foundation, Region Bretagne, Lorient Agglomeration, Serge Ferrari, World Courier, and KAUST. The global sampling effort was enabled by countless scientists and crew who sampled aboard the Tara from 2009-2013, and we thank MERCATOR-CORIOLIS and ACRI-ST for providing daily satellite data during the expeditions. We are also grateful to the countries who graciously granted sampling permission. The authors declare that all data reported herein are fully and freely available from the date of publication, with no restrictions, and that all of the analyses, publications, and ownership of data are free from legal entanglement or restriction by the various nations whose waters the Tara Oceans expeditions sampled in. This article is contribution number XX of Tara Oceans.

%\section*{Author contributions statement}

%Must include all authors, identified by initials, for example:
%A.A. conceived the experiment(s), A.A. and B.A. conducted the experiment(s), C.A. and D.A. analysed the results. All authors reviewed the manuscript. 

%\section*{Competing interests} (mandatory statement)

%The corresponding author is responsible for providing a \href{https://www.nature.com/sdata/policies/editorial-and-publishing-policies#competing}{competing interests statement} on behalf of all authors of the paper. This statement must be included in the submitted article file.



\end{document}
